script/sheet/armor.js

```diff
@@ -4,6 +4,28 @@ export class ForbiddenLandsArmorSheet extends ForbiddenLandsItemSheet {
   static get defaultOptions() {
     return mergeObject(super.defaultOptions, {
       template: "systems/forbidden-lands/model/armor.html",
+      tabs: [
+        {
+          navSelector: ".sheet-tabs",
+          contentSelector: ".sheet-body",
+          initial: "attributes",
+        },
+      ],
     });
   }
+
+  computeQuality(data) {
+    data.artifact = data.data.artifactBonus !== "";
+  }
+
+  getData() {
+    const data = super.getData();
+    this.computeQuality(data);
+    return data;
+  }
+
+  async _renderInner(data, options) {
```

Comment:
I don't think we need this method in either of these classes. It should already be covered by `ForbiddenLandsItemSheet._renderInner`, right?

Reply:
HA! Good catch, and actually… I think it's my fault. 😅 
I've done that in 03d2da3 for the weapon sheet, and I don't remember why. It does seem pointless.

Also: same thing in `gear.js`.

Reply:
Good catch. I have still not had the time to actually sit down and understand this mess. I copied the blocks into each model. But yeah noticed the base item.js had a render inner too, assumed these additional ones were somehow necessary.